content/service-manuals.md

```diff
@@ -1,45 +0,0 @@
----
-title: Service Manuals for System76 machines
```

Comment:
@ahoneybun did you mean to delete this?

Reply:
Yep compressing into one article so it's easier to keep up to date. I need to update pdf links in the next PR.